src/binding.rs

```diff
@@ -75,6 +75,16 @@ pub enum Binding {
     ///
     /// Useful for expressing empty bindings in [presets](crate::preset).
     None,
+    /// Any key, mouse button, or gamepad button, captured as [`ActionValue::Bool`].
+    /// If used with a context that selects an input device, such as
+    /// [`GamepadDevice::Single`], it will only capture inputs from that device.
+    ///
+    /// When activated by the indicated inputs this binding will consume all remaining
+    /// inputs. Any actions at a lower priority, whether in the same context or lower
+    /// priority contexts, will not activate. If you want this binding to activate
+    /// prior to bindings that include modifier keys then it needs to be placed in a
+    /// higher priority context.
```

Comment:
How about to rewrite it like this? Because other bindings, such as mouse movement, may still trigger other actions.
```suggestion
    /// If [`ActionSettings::consume_input`] is set, this binding consumes all button
    /// inputs, not just the one that activated it. To have an action with this binding
    /// evaluated first, place it in a higher-priority context.
```